app/components/PartialChallengeDetailsView.js

```diff
@@ -91,15 +66,51 @@ PartialChallengeDetailsView.propTypes = {
 }
 
 const styles = StyleSheet.create({
```

Comment:
I've already been setting up the common css file. It's how I'm doing it on my branch. I figured everyone can do it their own way until mine gets in and then I can go back through and refactor everything so it's pointing to the common file.